webapp/src/components/ImageEditor/ImageCanvas.tsx

```diff
@@ -296,9 +296,10 @@ class ImageCanvasImpl extends React.Component<ImageCanvasProps, {}> implements G
 
         if (!imageData) {
             const textData = ev.clipboardData.getData("text/plain");
-            // text data 'looks like' an image
-            if (/^\s*img`[\s\da-f.#tngrpoyw]+`\s*$/im.test(textData)) {
-                imageData = textData;
+            // text data contains string that 'looks like' an image
+            const res = /img(`|\(""")[\s\da-f.#tngrpoyw]+(`|"""\))/im.exec(textData);
```

Comment:
we also support capital A-F

Reply:
that and the random supported capitals from `tngrpoyw` are allowed with the `i` flag ~